lib/dialects/abstract/query-generator.js

```diff
@@ -388,13 +388,16 @@ const QueryGenerator = {
    @private
   */
   arithmeticQuery(operator, tableName, attrValueHash, where, options, attributes) {
+    options = options || {};
+    _.defaults(options, this.options, { returning: true });
+
```

Comment:
Not sure if this should be wrapped in conditional. I figured since the returning flag is unused by arithmetic queries other dialects, it wouldn't have a negative impact.

Reply:
I think its fine, in the future, if a dialect is added that has support for it or if eg. mysql gets support for it, it will be all set.

Reply:
Why you need `this.options` ?

Reply:
Good question. It probably doesn't. I was mostly following existing patterns, and I was unable to find why other methods map defaults from `this.options`. I figured it was a gap in my own understanding and that there may be cases where options get attached to the query generator object which would need to get accounted for. I'm unaware where/how that would happen though, so I figured better safe than sorry.

Reply:
Remove `this.options` then

Reply:
Done!